language/move-lang/tests/move_check/parser/struct_type_trailing_comma.move

```diff
@@ -0,0 +1,3 @@
+module M {
+    struct S<T1, T2,> { } // Test a trailing comma in the type parameters
```

Comment:
Another one that we shouldn't allow

Reply:
As discussed in https://github.com/libra/libra/pull/1973, we will continue to allow this.